packages/tools/pie-wrapper-react/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@justeattakeaway/pie-wrapper-react",
+  "packageManager": "yarn@3.5.0",
+  "version": "0.1.0",
+  "description": "Common react wrappers for use for PIE Project",
+  "main": "dist/react-gen-wrapper.js",
+  "license": "Apache-2.0",
+  "type": "module",
+  "keywords": [
+    "PIE"
+  ],
+  "author": "",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "wrapper-react:add": "yarn cem analyze && node ./scripts/add-react-wrapper.js",
+    "wrapper-react:remove": "node ./scripts/remove-react-wrapper.js",
+    "test": "run jest"
+  },
+  "jest": {
+    "testPathIgnorePatterns": [
+      "__tests__/mocks/"
+    ]
+  },
+  "devDependencies": {
+    "@custom-elements-manifest/analyzer": "0.8.0",
+    "@lit-labs/react": "1.1.1",
+    "cem-plugin-module-file-extensions": "0.0.5"
+  },
+  "dependencies": {
```

Comment:
All of these should be devDeps I think.

Can you try removing `jest` from here, and updating the root to 29.5.0 - https://github.com/justeattakeaway/pie/blob/main/package.json#L83

Ideally we want a single `jest` version for the entire monorepo.

@babel/preset-env can also be removed, as we define it at the root - https://github.com/justeattakeaway/pie/blob/main/package.json#L55